terms/variableName

```diff
@@ -1,9 +1,22 @@
 {
     "@context": {
-        "rdfs": "http://www.w3.org/2000/01/rdf-schema#"
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "schema": "http://schema.org/",
+        "reproschema": "https://raw.githubusercontent.com/ReproNim/reproschema/master/schemas/",
+	    "reproterms": "https://raw.githubusercontent.com/ReproNim/reproschema/master/terms/"
     },
     "@id": "variableName",
-    "@type": "rdfs:property",
-    "rdfs:comment": "An element to describe the variable name used.",
-    "rdfs:label": "variableName"
+    "@type": "rdf:Property",
+    "rdfs:comment": "The name used to represent an item.",
+    "rdfs:label": "variableName",
+    "schema:domainIncludes": {
+        "@id": "reproschema:ScoringLogic"
+    },
+    "schema:rangeIncludes": {
+        "@id": "schema:Text"
+    },
+    "schema:sameAs": {
```

Comment:
isn't this just pointing to itself?

Reply:
yes. that's how I had seen it in some of the nodes in `schema.org`. But I don't see it anymore in their latest schema version. I didn't quite understand it to be honest. I'll remove it then?

Reply:
deleted it.